src/live.asp.net/Services/YouTubeShowsService.cs

```diff
@@ -22,6 +25,8 @@ public class YouTubeShowsService : IShowsService
     {
         public const string CacheKey = nameof(YouTubeShowsService);
 
+        public int ResultsCount { get; } = 3 * 8;
```

Comment:
actually, I was wondering if we should get the *results count* as a parameter for the GetRecordedShowsAsync method, or even better store it in `appSettings` so it wouldn't be bound to the `YouTubeShowsService` implementation.